src/views/Home/Home.test.jsx

```diff
@@ -9,7 +11,34 @@ const user = {
   motto: 'Res Non Verba',
   color: 'crimson',
 }
+test(' making usre the user have all the right properties', () => {
+  render(<Home user={user} />)
 
-test('Should render the user profile', () => {
+  expect(user).toHaveProperty('id')
+  expect(user).toHaveProperty('name')
+  expect(user).toHaveProperty('avatar')
+  expect(user).toHaveProperty('header')
+  expect(user).toHaveProperty('likes')
+  expect(user).toHaveProperty('motto')
+  expect(user).toHaveProperty('color')
+})
+
+test('Should render the user profile', async () => {
+  render(<Home user={user} />)
+  const { name, likes, motto, color } = user
+
+  const headingName = await screen.findByRole('heading', { name })
+  const img = await screen.findByAltText('header')
```

Comment:
Only the first one of these need to be `async` with `findBy`. The rest can just use `getBy`